scripts/codes/opss-scheme-membership-sizeband.json

```diff
@@ -0,0 +1,7 @@
+{"_id":"dad03beb-f20f-414f-802f-477d89c75f6a","code":"all-sizebands","label":"All sizebands","links":{"code_list":{"id":"opss-scheme-membership-sizeband","href":"http://localhost:22400/code-lists/opss-scheme-membership-sizeband"},"self":{"id":"all-sizebands","href":"http://localhost:22400/code-lists/opss-scheme-membership-sizeband/codes/all-sizebands"}}}
+{"_id":"a931c3aa-6ae2-4717-af98-cd9092f71439","code":"5,000-to-9,999","label":"5,000 to 9,999","links":{"code_list":{"id":"opss-scheme-membership-sizeband","href":"http://localhost:22400/code-lists/opss-scheme-membership-sizeband"},"self":{"id":"5,000-to-9,999","href":"http://localhost:22400/code-lists/opss-scheme-membership-sizeband/codes/5,000-to-9,999"}}}
```

Comment:
Can we remove the commas from the codes please? In the labels they're fine. Just not sure commas will be super usable in the URLs later

Reply:
This comment can be ignored - we probably dont want commas long term but the data has been loaded with commas so we need to be able to link it up now